src/app/account/invitation/[inviteId]/account-form.tsx

```diff
@@ -28,27 +29,60 @@ const Success: FC = () => {
     )
 }
 
-const formSchema = z.object({
-    firstName: z.string().trim().nonempty('cannot be left blank'),
-    lastName: z.string().trim().nonempty('cannot be left blank'),
-    password: z.string().trim().min(8, 'must be at least 8 characters'),
-})
+const formSchema = z
+    .object({
+        firstName: z.string().min(2, 'Name must be 2-50 characters').max(50, 'Name must be 2-50 characters'),
+        lastName: z.string().min(2, 'Name must be 2-50 characters').max(50, 'Name must be 2-50 characters'),
+        password: z
+            .string()
+            .min(8, '8 character minimum')
+            .max(64)
+            .refine((val) => /[0-9]/.test(val), 'One number')
+            .refine((val) => /[A-Z]/.test(val), 'One uppercase letter')
+            .refine((val) => /[^A-Za-z0-9]/.test(val), 'One special character *&^'),
+        confirmPassword: z.string(),
+    })
+    .superRefine(({ confirmPassword, password }, ctx) => {
+        if (confirmPassword !== password) {
+            ctx.addIssue({
+                code: 'custom',
+                message: 'Passwords do not match. Please re-enter them.',
+                path: ['confirmPassword'],
+            })
+        }
+    })
```

Comment:
can this share the scheme defined here?
https://github.com/safeinsights/management-app/blob/main/src/app/account/reset-password/pending-reset.tsx#L16-L25